README.md

```diff
@@ -2,145 +2,27 @@
-      interval: 20s
-      start_period: 10s
-      timeout: 10s
-      retries: 3
-
-  nginx:
-    build: compose
-    ports:
-      - 8001:80
-    volumes:
-      - ./nginx_site.conf:/etc/nginx/conf.d/default.conf:ro
-      - public:/opt/kimai/public:ro
-    restart: unless-stopped
-    depends_on:
-      - kimai
-    healthcheck:
-      test:  wget --spider http://nginx/health || exit 1
-      interval: 20s
-      start_period: 10s
-      timeout: 10s
-      retries: 3
-
-  kimai:
-    image: kimai/kimai2:fpm-alpine-1.8-prod
-    environment:
-      - APP_ENV=prod
-      - TRUSTED_HOSTS=localhost
-      - ADMINMAIL=admin@kimai.local
-      - ADMINPASS=changemeplease
-    volumes:
-      - public:/opt/kimai/public
-      - var:/opt/kimai/var
-      # - ./ldap.conf:/etc/openldap/ldap.conf:z
-      # - ./ROOT-CA.pem:/etc/ssl/certs/ROOT-CA.pem:z
-    restart: unless-stopped
-    healthcheck:
-      test: wget --spider http://nginx || exit 1
-      interval: 20s
-      start_period: 10s
-      timeout: 10s
-      retries: 3
-
-  postfix:
-    image: catatnight/postfix:latest
-    environment:
-      maildomain: kimai.local
-      smtp_user: kimai:kimai
-    restart: unless-stopped
-
-volumes:
-    var:
-    public:
-```
-
-## Permissions
-
-If you are mounting the code base into the container (`-v $PWD/kimai:/opt/kimai`) then you will need to fix the permissions on the var folder.
-
-```bash
-docker exec --user root CONTAINER_NAME chown -R www-data:www-data /opt/kimai/var
-```
-
-or
-
-```bash
-docker-compose --user root exec CONTAINER_NAME chown -R www-data:www-data /opt/kimai/var
-```
-
-## Runtime Arguments
-
-The following settings can set at runtime:
-
-Kimai/symfony core settings, see the symfony and kimai docs for more info on these.
+Run the latest master version of Kimai in production mode using a bundled DB.  **This is not suitable for production use**:
 
-```bash
-DATABASE_URL=sqlite:///%kernel.project_dir%/var/data/kimai.sqlite
-APP_SECRET=change_this_to_something_unique
-TRUSTED_PROXIES=nginx,localhost,127.0.0.1
-TRUSTED_HOSTS=nginx,localhost,127.0.0.1
-MAILER_FROM=kimai@example.com
-MAILER_URL=null://localhost
-```
+    docker run --rm -ti -p 8001:8001 --name kimai2 kimai/kimai2
 
-Start up values:
+Create an admin user in the new running docker:
 
-If set then these values will try and create a new admin user.
+    docker exec kimai2 /opt/kimai/bin/console kimai:create-user admin admin@example.com ROLE_SUPER_ADMIN
 
-```bash
-ADMINPASS=
-ADMINMAIL=
-```
+This docker transient and will disappear when you stop the container.
 
-## NGINX and proxying
+## Documentaion
 
-While outside the direct responsibility of this project we get a lot of issues reported that relate to proxying with NGINX into the FPM container.
-Note that you will need to set the name of your NGINX container to be in the list of TRUSTED_HOSTS when you start the kimai container.
+ * [Starting a dev instance](docs/dev-instance.md#dev-instances)
```

Comment:
This indentations violates the markdownlint rule [MD007 - Unordered list indentation](https://github.com/DavidAnson/markdownlint/blob/v0.20.4/doc/Rules.md#md007).